v19.2/unsplit-at.md

```diff
@@ -0,0 +1,192 @@
+---
+title: UNSPLIT AT
+summary: The UNSPLIT AT statement reverts a manual key-value layer range split at a specified row in a table or index.
+toc: true
+---
+
+The `UNSPLIT AT` [statement](sql-statements.html) removes a [split enforcement](split-at.html) on a key-value layer range [split](architecture/distribution-layer.html#range-splits), at a specified row in a table or index.
+
+## Synopsis
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/unsplit_table_at.html %}
+</div>
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/unsplit_index_at.html %}
+</div>
+
+## Required privileges
+
+The user must have the `INSERT` [privilege](authorization.html#assign-privileges) on the table or index.
+
+## Parameters
+
+ Parameter | Description
+-----------|-------------
+ `table_name` | The name of the table that should be unsplit.
+ `table_name @ index_name`<br>`standalone_index_name` | The name of the index that should be unsplit.
+ `select_stmt` | A [selection query](selection-queries.html) that produces one or more rows at which to unsplit the table or index.
+
+## Viewing schema changes
+
+{% include {{ page.version.version }}/misc/schema-change-view-job.md %}
+
+## Examples
+
+{% include {{page.version.version}}/sql/movr-statements-nodes.md %}
+
+### Unsplit a table
+
+The `crdb_internal.ranges` table contains information about ranges in your CockroachDB cluster. At this point, just one range contains the data in the `users` table:
```

Comment:
I would reword the last clause slightly to "At this point, the data in the `users` table can be stored in just one range".  The table is the more important "thing" to a SQL user while the range is merely there in service of it.